### ISSUE TITLE
conf.js in PM2 0.14.6

### ISSUE THREAD

**Issue by Artazor** [Tjatse/pm2-gui#27]
Looks like this file is not used by PM2 0.14+ however pm2-gui relies on it.
I have continuous deployment via GitHub WebHooks + PM2 (which is reboot resistant and resurrects all processes), so my PM2 surely works and is configured. However in my setup the file 'config.js' in `~/.pm2/conf.js` is empty, nevertheless it's present.

I've seen the following line in the code `require(pm2Config)(options)` it means that there should be something like 
```javascript
module.exports = function(options) {
     return {} // ???
}
```
I've put the code above into `conf.js` and pm2-gui at least started and became availble via http, however it shows that there is no running processes.

Any idea?
Is something wrong with my pm2?

**Comment by Tjatse** [comment]
`conf.js` is still in using, `pm2-gui` depends on `PM2` and reads configurations of `PM2` from `conf.js`.
Please make sure that you `PM2` daemon is spawned before starting `pm2-gui`, or restart `pm2-gui` after `PM2` re-spawned.

It seems that your `pm2-gui` instance can not locate the PM2 root, you can manually setting it by `pm2-gui set pm2 /var/path/to/.pm2`.

If the issue still can not be solved, tell me your platform, node version, e.g.:
```
node: 0.12.6
platform: CentOS 7.1 
```

**Comment by Averelll** [comment]
I run into exactly the same issue on Debian
node: 0.12.4
pm2: 0.14.5
running on Debian Jessie. 
~/.pm2/conf.js is present, but has 0 size.

**Comment by Tjatse** [comment]
So it means that your PM2 has been spawned yet, but `~/.pm2/conf.js` is an empty file? WEIRD!!!

**Comment by Averelll** [comment]
yeah, it is. I do run as root, I will check if somehow my conf.js ended up at a different location

**Comment by Averelll** [comment]
so starting pm2 as a different user does indeed create a valide conf.js. Copying this to ./root/.pm2/conf.js makes it work. Seems to be quite a static file. Interesting that running under root doesn't create the file.

**Comment by Tjatse** [comment]
Yup, that's the problem, you must run both PM2 and pm2-gui with a same user.  'Cause pm2-gui dependents on PM2 (only `conf.js` of PM2).  :)

**Comment by Averelll** [comment]
Eh, no, then you misunderstood me :)
I run both pm2 and pm2-gui as root
root user does not have a conf.js.
I ran pm2 once under a different userid, just to check. Then it does create a conf.js
then I copied the conf.js from the other user to /root/.pm2/conf.js
When I then run pm2 and pm2-gui as root, it works.

**Comment by smiziara** [comment]
Same problem here trying to run on Windows.

The conf.js exists but its empty. PM2 Daemon is spawned.

Both process are running with the same windows account :(

**Comment by Tjatse** [comment]
Fixed @0.1.2!
